docs/content/contribute/review-pr.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Review a PR"
+weight: 11
+---
+
+# Review a PR
+
+This page provides guidelines for reviewing Magic Modules pull requests
+
+1. Read the pull request description and the linked issues to understand the context and check if the pull request actually resolves the issues.
+2. If the PR adds any new resource, ensure that the resource does not already exists in the [GA provider](https://github.com/hashicorp/terraform-provider-google) or [beta provider](https://github.com/hashicorp/terraform-provider-google-beta)
+1. Read through all the changes in the pull request, generated code in the downstreams and the API documentation to ensure that:
+   1. the resource schema added in the pull request matches the API structure
+   1. the features are added in the correct version 
+      * features only available in beta are not included in the GA google provider.
+      * features added to the GA provider are also included in the beta provider -- beta should be a strict superset of GA.
+   1. no [breaking changes]({{< ref "/develop/breaking-changes" >}}) are introduced without a valid justification.
+1. Check the tests to ensure that:
+   1. all related tests, including acceptance tests and unit tests, have been completed successfully. 
```

Comment:
I would be inclined to put this lower down- I've seen a few cases where folks have trusted tests because they were green, but that tested the wrong thing. I'd encourage something along the lines of "Verify the tests ensure that the linked issue was resolved, if any".

For example- a bugfix should test the bug (or explain why it's not feasible to do so in the description) and an enhancement should test the new behaviour(s).